lib/Target/SubstraitPB/Export.cpp

```diff
@@ -181,10 +182,13 @@ FailureOr<std::unique_ptr<Rel>> exportOperation(EmitOp op) {
 
 FailureOr<std::unique_ptr<Expression>>
 exportOperation(ExpressionOpInterface op) {
-  return llvm::TypeSwitch<Operation *, FailureOr<std::unique_ptr<Expression>>>(
-             op)
-      .Case<FieldReferenceOp, LiteralOp>(
-          [&](auto op) { return exportOperation(op); })
+  return llvm::TypeSwitch<ExpressionOpInterface,
```

Comment:
@ingomueller-net I have removed workaround.

Reply:
I think you have to rebase onto `main`. I can't approve to trigger CI currently.

Reply:
OK, there might not have been any conflicts before, but it's been a while. I will resolve the conflicts shortly.

Reply:
Thanks! CI is running now...